apps/studio/src/pages/home/library/MintSong.tsx

```diff
@@ -424,7 +424,9 @@ const MintSong = () => {
 
                     { isStepOneButtonVisible && (
                       <Button
-                        disabled={ !isVerified || !wallet }
+                        disabled={
+                          !isMintingInitiated && (!isVerified || !wallet)
```

Comment:
Was deciding if this conditional is too convoluted with all the `!`, was thinking of these alternatives:

1. Create a new reference `const isMintingFormVisible = !isMintingInitiated && isMinting`. Then replace it here and for the alerts above. The updated line would be: 
`disabled={ isMintingFormVisible && (!isVerified || !wallet)`

2.  This one might convolute even more but reversing the entire conditional and putting the `!` outside to basically state this is not disabled when this condition is met. 
`disabled={ !(isMintingInitiated || (isVerified && wallet)`